src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -62,14 +73,10 @@
  *
  * <pre>public final class TkDiscussion implements Take {
  *   &#64;Override
- *   public Response act(final Request req) {
- *     if (failed) {
- *       throw new RsFlash(
- *         new RsForward(),
- *         "can't save your post, sorry",
- *         java.util.logging.Level.SEVERE
- *       );
- *     }
+ *   public Response act(final Request req) throws IOException {
+ *     return new RsForward(
+ *         new RsFlash("can't save your post, sorry",
+ *         java.util.logging.Level.SEVERE));
```

Comment:
@essobedo please fix code sample according to http://www.yegor256.com/2014/10/23/paired-brackets-notation.html